tests/scenarios/node-builtins.test.ts

```diff
@@ -0,0 +1,28 @@
+import esbuild, { type BuildOptions } from 'esbuild';
+import { assertFileContent, buildAbsolutePath, createEsbuildConfig } from '../common/util';
+
+function createConfig(platform: 'node' | 'browser'): BuildOptions {
+  return createEsbuildConfig({
+    platform,
+    format: 'esm',
```

Comment:
```suggestion
    format: 'esm',
    outExtension: {
      '.js': '.mjs'
    },
```

This will fix the last 2 tests. Currently it outputs as `.js` which conflicts with line 18 and 26.